judge-d-agent/pom.xml

```diff
@@ -34,32 +37,32 @@
         <dependency>
             <groupId>io.github.openfeign</groupId>
             <artifactId>feign-core</artifactId>
-            <version>9.6.0</version>
+            <version>${feignVersion}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-openfeign-core</artifactId>
-            <version>2.0.1.RELEASE</version>
+            <version>${springCloudVersion}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-netflix-hystrix</artifactId>
-            <version>2.1.0.RELEASE</version>
+            <version>${hystrixVersion}</version>
```

Comment:
${springCloudVersion}

Reply:
Hystrix is no longer in active development, and is currently in maintenance mode so the version is different than other cloud dependencies, anyway, it is not used so I removed this dependency